rest/v9/oauth2.ts

```diff
@@ -0,0 +1,199 @@
+import type { Permissions, Snowflake } from '../../globals';
+import type { APIApplication, APIGuild, APIUser, APIWebhook, OAuth2Scopes } from '../../payloads/v9/index';
+
+/**
+ * https://discord.com/developers/docs/topics/oauth2#get-current-application-information
+ */
+export type RESTGetAPIOauth2CurrentApplicationResult = Omit<APIApplication, 'flags'>;
```

Comment:
For the sake of my sanity, please rename the `Oauth` to `OAuth` (notice the capital A). I will do this for API v8

Reply:
There's a couple places in Changelog and rest/common do you want those changed too?

Reply:
I mean I checked, there doesn't seem to be anything in rest/common? Can you link me? Also, ignore the changelog

Reply:
There's 3 here: https://github.com/discordjs/discord-api-types/blob/f0b2766d5b55bd9b8b8ba9c506a868dafcdca568/rest/common.ts#L80-L82

And one a little later, they are capitalized in the dev docs

Reply:
Oh fuck. Do those in API v9, I will quickly unfuck that in v8. Nice catch, thanks a bunch <3

Reply:
Wait, I'm blind, thought that was an import. Will fix